client/src/pages/PackageDetailsScreen/MatchCard/MatchCard.tsx

```diff
@@ -0,0 +1,48 @@
+import styles from './match-card.module.scss';
+import { Match, Package } from '@/models/package.model';
+import { romanize } from 'romans';
+import { MatchLabel } from './MatchLabel';
+import { MatchDetails } from './MatchDetails';
+
+interface MatchCardProps {
+    match: Match;
+    singlePackage: Package;
```

Comment:
```suggestion
    package: Package;
```
no need to mention that it is a singlePackage if it is from type of Package and not array of packages

Reply:
Identifier expected. 'package' is a reserved word in strict mode. Modules are automatically in strict mode